src/modules/services/logger.ts

```diff
@@ -0,0 +1,15 @@
+import Logger from '$lib/logger/logger';
+import { Service } from 'purplet';
+
+let logger: Logger;
+
+export default Service({
+  name: 'Logging',
+  async init() {
+    logger = new Logger(this.client, process.env.ADMIN_CHANNEL || '');
```

Comment:
With the .env typescripting (which I didn't know existed, so I learned something new!) I think we can remove the `|| ''`, couldn't we?

Reply:
no because logger expects a string and ADMIN_CHANNEL is `string | undefined`

Reply:
i have made channel optional in logger, thus removing the || ''